thyrel-web/src/components/room/book/BookStartAction.tsx

```diff
@@ -1,24 +1,28 @@
 import { usePlayerContext } from 'hooks/PlayerProvider';
 import StartButton from '../lobby/StartButton';
-//import { client } from 'api/client';
-//import Session from 'types/Session.type';
-//import { getToken } from 'api/player-provider';
-//import { useSnackbar } from 'notistack';
+import { client } from 'api/client';
+import { getToken } from 'api/player-provider';
+import { useSnackbar } from 'notistack';
 
 export default function BookStartAction() {
-  //const { enqueueSnackbar } = useSnackbar();
+  const { enqueueSnackbar } = useSnackbar();
   const { player } = usePlayerContext();
 
   function onStart() {
-    // TODO: When the NextAlbum() controller will be done.
-    // client<Session>('session', { method: 'POST', token: getToken() }).then(
-    //   () =>
-    //     enqueueSnackbar('Game successfully started 💪', { variant: 'success' }),
-    //   () =>
-    //     enqueueSnackbar('Sorry, an error occured 😕 [Session-POST]', {
-    //       variant: 'error',
-    //     }),
-    // );
+    client<null>('session/album/next', {
+      method: 'POST',
+      token: getToken(),
+    }).then(
+      () =>
+        enqueueSnackbar(
+          'Album successfully started, wait 3 secondes before starting 💪',
```

Comment:
pas de "e" en anglais

```suggestion
          'Album successfully started, wait 3 seconds before starting 💪',
```